### ISSUE TITLE
create new feedback survey 

### ISSUE THREAD

**Issue by lauradereynal** [mozilla/webmaker-core#627]
Goal: Create a new survey for Webmaker. 
When: Due for Thursday 10th September 

Things we want to learn: 
- How did you discover Webmaker (multiple choice)
- What did you think you could do with it ?
- What did you make on Webmaker ? 
- How would you rate your experience in general ? 1 to 10 (1 is lowest 10 is best)
- Can you explain your choice ?
- Do you think you will use Webmaker again ? 
- If yes, great, why? 
- If no, why not ?
- Can you tell us 3 things to improve for a better experience?

...

**Comment by lauradereynal** [comment]
@xmatthewx @HPaulJohnson 

Is it possible to add the survey in the play store? 
Maybe we could add it in our responses for anyone who leaves comments and reviews on the playstore. 

I'd love to have non-mozillians answering it, and for now, it's been a lot of community members because of the way we promoted this survey on Twitter etc.

**Comment by xmatthewx** [comment]
Yes. We could put it at the bottom of the description. You'd get a very particular slice of traffic opting-in. Is that useful to you?
